.github/workflows/build.yml

```diff
@@ -30,3 +30,19 @@ jobs:
           coverage
           .nyc_output
       if: ${{ failure() }}
+
+  e2e:
```

Comment:
`build` is not an accurate name for that CI job, there is no build step for cht-conf, it would be clearer if it was named `unit-tests`
with that said, I think it makes sense to run all tests in parallel like we do in cht-core and I would prefer to keep it this way if that makes sense to you too

Reply:
Ah good point!  Had not thought of it that way.  Makes sense. :+1: